### PR TITLE
Fix/nex 946/exam end url

### PR DESCRIPTION
**Pull request by mike-ionut-mihai-sandu-tao**
https://oat-sa.atlassian.net/browse/NEX-946
 - fixed the exam_end url

### CODE REVIEW

File: scripts/update/Updater.php [hunk 1]
```diff
@@ -29,10 +29,6 @@ class Updater extends common_ext_ExtensionUpdater
      */
     public function update($initialVersion)
     {
-        $this->skip('1.0.0', '1.0.1');
-
-        if ($this->isVersion('1.0.1')) {
-            $this->setVersion('1.0.2');
-        }
+        $this->skip('1.0.0', '1.0.3');
```
Comment (gabrielfs7) [inline scripts/update/Updater.php:32]: Must be 1.4, cause @boajer  branch is already 1.3

**Comment by gabrielfs7** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are

File: model/request/ProctorioExamUrlFactory.php [hunk 2]
```diff
@@ -28,20 +28,38 @@
 class ProctorioExamUrlFactory extends ConfigurableService
 {
     public const OPTION_BASE_URL = 'base_url';
+    private const DELIVERY_EXECUTION_TAKE_URI = '/taoDelivery/DeliveryServer/runDeliveryExecution';
```
Comment (boajer) [inline model/request/ProctorioExamUrlFactory.php:31]: Should we make is somehow aware of \oat\remoteProctoring\controller\DeliveryLaunch::getRedirectUrl, that is used  in `\oat\remoteProctoring\controller\DeliveryLaunch::launch`, now all urls looks hardcoded;

Same applyes for constants bellow. We can rely on url helpers etc, what do you think ?

Comment (mike-ionut-mihai-sandu-tao) [reply model/request/ProctorioExamUrlFactory.php:31]: added inside the improvements page ::
https://oat-sa.atlassian.net/wiki/spaces/NEX/pages/326468245/Proctorio+Improvements

**Pull request merged**